modules/uipath/src/backend/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as sdk from 'botpress/sdk'
+
+const onServerStarted = async (bp: typeof sdk) => {
+  const router = bp.http.createRouterForBot('uipath', {
+    checkAuthentication: false
+  })
+
+  router.post('/message', async (req, res) => {
+    try {
+      const body = req.body
+
+      await bp.events.replyToEvent(
+        {
+          channel: body.channel,
+          target: body.target,
+          botId: body.botId,
+          threadId: body.threadId
+        },
+        [body.message]
+      )
+
+      res.sendStatus(200)
+    } catch (err) {
+      res.status(400).send(err)
+    }
+  })
+}
+const onServerReady = async (bp: typeof sdk) => {}
```

Comment:
The API should be initialized in "onServerReady" since it's when the HTTP server is sure to be available. onServerStarted is mostly for db setup and other initializations. (also, they both are optional and can be omitted if empty)